README.md

```diff
@@ -31,7 +31,7 @@ Here's a snippet of what we intend to deliver in 2021 *(times/features/releases
-Unpackaged desktop apps<br>(WPF, Win32, WinForms, console) |    🔄 [Text rendering](https://docs.microsoft.com/windows/apps/project-reunion/dwritecore)<br>🔄 [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)<br>🔄 [Dynamic Dependencies](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/dynamicdependencies/DynamicDependencies.md)<br>🔄 [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111)  |  ✅ [WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3)<br>✅ [Text rendering](https://docs.microsoft.com/windows/apps/project-reunion/dwritecore)<br>✅ [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)<br>✅ [Dynamic Dependencies](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/dynamicdependencies/DynamicDependencies.md)<br>🔄 [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) <br>🔄 [Push Notifications](https://github.com/microsoft/ProjectReunion/issues/334) |
+Unpackaged desktop apps<br>(WPF, Win32, WinForms, console) |    🔄 [Text rendering](https://docs.microsoft.com/windows/apps/project-reunion/dwritecore)<br>🔄 [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)<br>🔄 [Dynamic Dependencies](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/dynamicdependencies/DynamicDependencies.md)<br>🔄 [https://github.com/microsoft/WindowsAppSDK/pull/1009App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111)  |  ✅ [WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3)<br>✅ [Text rendering](https://docs.microsoft.com/windows/apps/project-reunion/dwritecore)<br>✅ [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)<br>✅ [Dynamic Dependencies](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/dynamicdependencies/DynamicDependencies.md)<br>✅ [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) <br>🔄 [Push Notifications](https://github.com/microsoft/ProjectReunion/issues/334) |
```

Comment:
The link text for 0.8 App lifecycle has an extra github PR link:
![image](https://user-images.githubusercontent.com/16918354/124093200-35546f00-da8a-11eb-9c92-bbda466c7dd5.png)

Reply:
Thank you.